src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -118,6 +118,7 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
 
   toPreviousPage() {
     let caseEventData: CaseEventData = this.buildCaseEventData();
+    caseEventData.data = caseEventData.event_data;
```

Comment:
Can you put a comment in here explaining what this is for?

Reply:
ah yes